tuna/miopen/utils/json_to_sql.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE.
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+"""Utility module for parsing fin json results"""
+import functools
+from sqlalchemy.exc import OperationalError
+
+from tuna.utils.logger import setup_logger
+from tuna.dbBase.sql_alchemy import DbSession
+from tuna.utils.utility import SimpleDict
+from tuna.utils.db_utility import session_retry, gen_select_objs
+from tuna.utils.db_utility import gen_update_query, gen_insert_query
+from tuna.miopen.worker.fin_utils import get_fin_slv_status
+from tuna.miopen.utils.parsing import parse_pdb_key
+from tuna.miopen.db.solver import get_solver_ids
+
+LOGGER = setup_logger('parse_results')
+
+
+def __update_fdb_w_kernels(  #pylint: disable=too-many-arguments,too-many-locals
```

Comment:
file that contains all the fin results processing functions moved over from fin_builder/eval